submissions/kkkovalllchuk/html-css-popup/styles.css

```diff
@@ -0,0 +1,159 @@
+* {
+  margin: 0;
+  padding: 0;
+  color: black;
+}
+
+body {
+  font-family: "Gill Sans", sans-serif;
+  background: gray;
+  background-size: cover;
+  height: 100%;
+  overflow: hidden;
+}
+
+img {
+  max-width: 100%;
+  max-height: 40px;
+  border: solid 5px transparent;
+}
+
+.nav {
+  margin: 5px;
+  float: right;
+  display: flex;
+  align-items: center;
+  
+}
+
+
+
+.nav__list-item {
+  list-style: none;
+  margin: 10px 5px;
+  border: solid 1px transparent;
+}
+
+.menu {
+  display: flex;
+  background-color: rgb(182, 182, 182);
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+  border: solid 2px rgb(69, 69, 69);
+}
+
+.menu__popup {
+  display: block;
+  padding: 8px;
+}
+
+.popup-input,
+.more-input {
+  position: absolute;
+  z-index: -1;
+  opacity: 0;
+}
+
+.popup-label {
+  padding: 8px;
+  display: block;
+  width: 40px;
+  height: 40px;
+  cursor: pointer;
+  background: url("./img/icons8-поделиться.svg") no-repeat center;
+}
+
+.main {
+  position: relative;
+}
+.popup-main {
+  justify-content: center;
+  display: none;
+  flex-wrap: wrap;
+  position: absolute;
+  right: -50%;
+  margin-top: 20px;
+  align-self: flex-end;
+  width: 350px;
+  height: 280px;
+  padding: 10px;
+  overflow-y: scroll;
+  z-index: 1;
+  background-color: white;
+  opacity: 0.7;
+}
+.nav__list-link,
+.popup-item
+ {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  border: solid 10px transparent;
+}
+
+.popup-item {
+  margin: 4px;
+  padding: 5px;
+  height: 80px;
+  width: 24%;
+}
+
+.popup-item > img {
```

Comment:
Avoid styling by HTML tags except if you add base styles. [Reason](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)
Also, avoid using too specific CSS selectors. [Read more](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)